tap_wikipedia/models/types/image_url.py

```diff
@@ -0,0 +1,8 @@
+from typing import Annotated
+
+from pydantic import Field
+
+# A tiny type for a Wikipedia article's ImageURL.
+ImageUrl = Annotated[
```

Comment:
Pydantic has a bunch of built-in Annotated's you may want to reuse, like AnyUrl here: https://docs.pydantic.dev/latest/api/networks/#pydantic.networks.AnyUrl

Reply:
You don't necessarily need a unique tiny type for every field. The tiny types should be for unique semantics e.g., "URL" or "non-blank string" or "UuidIdentifier".

Reply:
I have removed the unnecessary tiny types from the project.